jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -91,6 +92,14 @@
 	private static Map<Class<?>, Object> setSMCallers;
 	/*[ENDIF] JAVA_SPEC_VERSION >= 17 */
 
+	/*[IF JAVA_SPEC_VERSION >= 12] */
```

Comment:
```suggestion
	/*[IF JAVA_SPEC_VERSION > 11] */
```
I recommend LTS level.

Reply:
Updated.